modules/addingBook.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable semi */
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] Please remove disable lines for `eslint` and Fix all linters issues.
[Hint]: for line break error that happens because you are a windows user you can simply fix it by clicking on ` CRLF ` on the right bottom of your VS code like the image below then choose `LF` line break instead.
![image](https://user-images.githubusercontent.com/68796232/172885684-feca74d8-1515-4409-a07b-5e824d9dda69.png)
![image](https://user-images.githubusercontent.com/68796232/172885715-72d5ae28-893f-4d31-a448-3c031af52791.png)

For other linters errors you can fix it using `npx eslint . --fix`